src/ReDate.re

```diff
@@ -676,3 +676,102 @@ let eachDayOfIntervalList = interval =>
       interval |> makeIntervalDay,
     )
   );
+
+/* ——[ISO Week helpers]——————— */
+
+let endOfISOWeek = date => endOfWeek(~weekStartsOn=Monday, date);
+
+let isSameISOWeek = (fst, snd) => isSameWeek(~weekStartsOn=Monday, fst, snd);
+
+let isThisISOWeek = date => isSameISOWeek(Js.Date.make(), date);
+
+let lastDayOfISOWeek = date => lastDayOfWeek(~weekStartsOn=Monday, date);
+
+let startOfISOWeek = date => startOfWeek(~weekStartsOn=Monday, date);
+
+let differenceInCalendarISOWeeks = (fst, snd) => {
+  let startOfFstISOWeek = startOfISOWeek(fst);
+  let startOfSndISOWeek = startOfISOWeek(snd);
+
+  let timestampFst =
+    Js.Date.getTime(startOfFstISOWeek)
+    -. Internal.getTimezoneOffsetInMilliseconds(startOfFstISOWeek);
+  let timestampSnd =
+    Js.Date.getTime(startOfSndISOWeek)
+    -. Internal.getTimezoneOffsetInMilliseconds(startOfSndISOWeek);
+
+  let milliseconds_in_week = 604800000.;
```

Comment:
could you use `Milliseconds.week`? 
https://github.com/mobily/re-date/blob/5a0c9d8b6c01a821fc88b5e6485713b0a105d95a/src/ReDate.re#L48